apps/vh/base/nonprod-role.yaml

```
@@ -12,6 +12,9 @@ rules:
 - apiGroups: ['helm.fluxcd.io']
   resources: ['helmreleases']
   verbs: ['delete']
+- apiGroups: ['v2beta1.helm.fluxcd.io']
```

this definitely right and not?
```suggestion
- apiGroups: ['helm.fluxcd.io']
```

---

I was looking at the output of `k get apiservices.apiregistration.k8s.io -A`. There's v1.helm.fluxcd.io and v2beta1.helm.fluxcd.io. Will helm.fluxcd.io do it for both?

---

I think so

---

I've reverted it there